accepted/0000-sle-micro-reboot-action.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: SLE Micro - reboot action
+- Start Date: 2021-09-29
+
+# Summary
+[summary]: #summary
+
+Provide a suitable approach to handle reboot action in SLE Micro
+
+# Motivation
+Transactional systems, like SLE micro, use `transactional-update` to perform management actions including reboot. Transactional update itself supports several different reboot methods, configurable via the REBOOT_METHOD configuration option in `transactional-update.conf`. In order to put Suma in control of the reboot of its managed systems, it is necessary to configure transactional systems in a proper way and carry out the reboot action according to their particularities.
```

Comment:
About the 1st sentence. T-u is not required for reboot- it's possible to reboot without t-u.

Reply:
I updated the text to make it clearer, thank you.